tests/functions/test_flag_nulls.py

```diff
@@ -0,0 +1,72 @@
+import pandas as pd
+import pytest
+from pandas.testing import assert_frame_equal
+
+TEST_DF = pd.DataFrame({"a": [1, 2, None, 4], "b": [5.0, None, 7.0, 8.0]})
```

Comment:
Wondering if you might be open to putting this in `conftest.py` as a fixture? 

We also have a `nulldf` fixture inside `conftest.py`, which may come in handy, and reduce code duplication.